dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -149,18 +150,18 @@ public class PagedDoraWorker extends AbstractWorker implements DoraWorker {
   // for now Dora Worker does not support Alluxio <-> UFS mapping,
   // and assumes all UFS paths belong to the same UFS.
   private static final int MOUNT_POINT = 1;
-  private final Closer mResourceCloser = Closer.create();
+  protected final Closer mResourceCloser = Closer.create();
   // TODO(lucy) change to string typed once membership manager got enabled by default
   private final AtomicReference<WorkerIdentity> mWorkerId;
   protected final CacheManager mCacheManager;
-  protected final DoraUfsManager mUfsManager;
-  private final DoraMetaManager mMetaManager;
+  protected final UfsManager mUfsManager;
```

Comment:
Use interface here since we are using dependency injection. If one method is not defined on the interface (in the concrete class), I'm moving the signature to the interface level.